lib/xcflushd/threading.rb

```diff
@@ -3,10 +3,8 @@
 
 module Xcflushd
   module Threading
-    def self.default_threads_value
-      cpus = Concurrent.processor_count
-      # default thread pool minimum is zero
-      return 0, cpus * 4
+    def self.default_threads
+      Concurrent.processor_count
```

Comment:
I didn't mean to change the default values. I just didn't notice that `* 4` 🤦‍♂️